docs/aerodromes/classd/Moorabbin.md

```diff
@@ -21,49 +24,81 @@ MB ADC is responsible for the Class D airspace in the MB CTR `SFC` to `A025`.
+2. Runway 17L/17R  
+3. Runways 13/31 (equal preference)
 
-When operationally required, runway 04 or 22 may be used, but consideration must be given to it's limited length.
+Crosswind operations are limited to **10KT**, and tailwind use is discouraged unless required.
+
+Runway 04/22 may only be used operationally and is **not available for circuit training**. Consideration must be given to it's limited length.
+
+---
 
 ## VFR Inbound Procedures
 
-| VFR Approach Point | RWY 13  | RWY 17 | RWY 31  | RWY 35 |
-| ----------------| --------- | ---------- | --------- | ---------- |
-| BTO | Join final 13R | Join base 17R | Join downwind 31L | Join oblique downwind 35L |
-| BAW  | Join base 13R | Join base 17R | Join oblique downwind 31L | Join base 35L |
-| CARR | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
-| GMH  | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
-| ACE  | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
+Aircraft should report at the standard VFR entry points:  
+- **BTO** (to the west)  
+- **BAW** (to the north)  
+- **CARR** (to the southeast)  
+- **GMH** (to the east)  
+- **ACE** (to the south)  
+- **SHOAL** (to the southwest)
+
+| Entry Point | RWY 13 | RWY 17 | RWY 31 | RWY 35 |
+|-------------|--------|--------|--------|--------|
+| BTO         | Join final 13R | Join base 17R | Join downwind 31L | Join oblique downwind 35L |
+| BAW         | Join base 13R  | Join base 17R | Join oblique downwind 31L | Join base 35L |
+| CARR        | Join downwind 13R | Join downwind 17R | Join oblique base 31L | Join final 35L |
+| GMH         | Join downwind 13L | Join base 17L | Join oblique base 31R | Join base 35R |
+| ACE         | Join base 13L | Join oblique base 17L | Join base 31R | Join oblique downwind 35R |
+| SHOAL       | Join base 13L | Join base 17L | Join base 31R | Join base 35R |
+
+CTR Entry Altitude: `A010`.
+
+---
 
 ## Departures into CTA
-The overlying Class C airspace extends down to A025 to the north and A045 to the south. Aircraft planned into the overlying Class C airspace shall be issued an airways clearance by **MB ADC** and assigned `A050` or `RFL` if lower. These aircraft must be Next coordinated to the relevant TCU controller, who will assess the current traffic picture and provide a release when able. During times of peak TMA traffic, there may be extensive delays for departure and pilots may elect to depart directly into class G. See [Departure Coordination](#departures) for coordination requirements.
+
+The overlying Class C airspace extends down to `A025` to the north and `A045` to the south.
+
+- Aircraft departing into Class C must be cleared to `A050` or `RFL` (if lower) by **MB ADC**.  
+- Coordination with TCU for release is required before issuing a takeoff clearance.  
+- VFR aircraft departing into Class G must advise intentions with their "Ready" call and will be cleared accordingly.
```

Comment:
Can probably remove, since this section details departures into CTA

Reply:
Western triangle and eastern grass are parts of the runway/active environment and include grass areas . Essentially they land in the unused area not used by current traffic. Similar to how at Essendon they have an eastern grass which is a portion of the aerodrome area for helicopters to train. 

So it's part of the adc controller area. I'm doing a diagram based on vatsys. I have some Mac (Moorabbin corporation) materials I am converting to vatsys based diagrams

Reply:
Yeah can do, this is just what was there made into more bullet point form